types/trigger.d.ts

```diff
@@ -1,19 +1,17 @@
 import { Lang, NonEnLang } from '../resources/languages';
 import { TimelineReplacement, TimelineStyle } from '../ui/raidboss/timeline';
 import { RaidbossData } from './data';
-
-export interface BaseRegExp<T> extends RegExp {
-  groups?: {
-    [s in T]?: string;
-  };
-}
+import { CactbotBaseRegExp, TriggerTypes } from './net_trigger';
+import { NetAllMatches, Matches as MatchesAny } from '../types/net_matches';
```

Comment:
I think there is only one use of `MatchesAny` so no need to import it with a separate name.

Can you leave a TODO to remove `Matches<Params>` below? (I assume that's the followup here?, to switch everything to `NetAllMatches`?)

Reply:
That would conflict with the `export type Matches<Params>` below, that's why the rename in the import. Something to fix with cleanup in followup.